examples/language/bert/test_ci.sh

```diff
@@ -6,3 +6,5 @@ pip install -r requirements.txt
 for plugin in "torch_ddp" "torch_ddp_fp16" "gemini" "low_level_zero"; do
    torchrun --standalone --nproc_per_node 4  finetune.py --target_f1 0.86 --plugin $plugin --model_type "bert"
 done
+
+torchrun --standalone --nproc_per_node=1 shardformer_benchmark.py
```

Comment:
In order to test the sharding, plaese use more than 1 GPU and set a small number of iteration so that it can finish running within 1-2 minutes.